lib/uname.js

```diff
@@ -0,0 +1,43 @@
+var getUname = require('node-uname')
```

Comment:
Is not possible to get that info from the Node.js built-ins instead of from a compiled module?